src/Share.tsx

```diff
@@ -31,6 +32,7 @@ function parseStandardParams(): EventQS {
     timezone: params.get("tz") || "",
     isOnline: typeof params.get("o") === "string",
     isAllDay: typeof params.get("a") === "string",
+    creatorEmail: params.get("ce") || "",
```

Comment:
I'd say "e" is enough. (First come, first serve :P)
Also "email" instead of "creatorEmail"